luigi/parameter.py

```diff
@@ -219,6 +219,8 @@ def parse(self, x):
         :param str x: the value to parse.
         :return: the parsed value.
         """
+        if not isinstance(x, basestring) or not isinstance(x, str):
```

Comment:
Or apparently all the python 3 tests will fail because they don't know what `basestring` is.

How should one go about this condition? I need `if not isinstance(x, basestring)` for python2 and `if not isinstance(x, str)` for python3.